suites/nautilus/upgrades/tier_1_upgrade_rpm_upgrade.yaml

```diff
@@ -0,0 +1,131 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier1_upgrade_container.yaml
```

Comment:
Would it be more compact to use the name as tier_1_upgrade_rpm.yaml instead of tier_1_upgrade_rpm_upgrade.yaml.